exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -685,6 +685,18 @@ private ExecConstants() {
   public static final String WEB_LOGS_MAX_LINES = "web.logs.max_lines";
   public static final OptionValidator WEB_LOGS_MAX_LINES_VALIDATOR = new PositiveLongValidator(WEB_LOGS_MAX_LINES, Integer.MAX_VALUE, null);
 
+  public static final String WEB_TIMESTAMP_DISPLAY_FORMAT = "web.timestamp.display_format";
+  public static final OptionValidator WEB_TIMESTAMP_DISPLAY_FORMAT_VALIDATOR = new StringValidator(WEB_TIMESTAMP_DISPLAY_FORMAT,
+      new OptionDescription("Display format template for timestamp. If empty then the default formatting will be used."));
```

Comment:
Maybe use can mention what formatting can be applied.

Reply:
Updated the descriptions.